tests/E2E/test/campaigns/full/01_orders/06_delete_shopping_carts.js

```diff
@@ -11,13 +11,15 @@ const {CheckoutOrderPage} = require('../../../selectors/FO/order_page');
 const {OrderPage, ShoppingCart} = require('../../../selectors/BO/order');
 const {CatalogPage} = require('../../../selectors/BO/catalogpage/index');
 const commonOrder = require('../../common_scenarios/order');
+const welcomeScenarios = require('../../common_scenarios/welcome');
 let promise = Promise.resolve();
 
 scenario('Delete shopping carts', () => {
   scenario('Login in the Back Office', client => {
     test('should open the browser', () => client.open());
     test('should login successfully in the Back Office', () => client.signInBO(AccessPageBO));
   }, 'order');
+  welcomeScenarios.findAndCloseWelcomeModal();
```

Comment:
Should it be better to add a beforeEach scenario to be sure the welcome scenarios is executed?

Reply:
Problem is we can only use this method when we're connecting **for the first time** in the BO, and that can happen anywhere in tests (sometime at the beginning, sometimes in the middle). Some tests don't even connect to the BO.
We also prefer to have it independent, since we want to do this once per test (it's not necessary to waste time by adding it on every BO connection after the first one).
But I recon it's a little bit tedious.

I added this because we thought it would be better to have each test to be independent.